components/Root.tsx

```diff
@@ -29,6 +29,12 @@ import { ClientSideGetInitialPropsContext } from '../global/utils/pages/types';
 import Head from './Head';
 import defaultTheme from './theme';
 import SystemAlerts from './SystemAlerts';
+import React from 'react';
+
+/**
+ * portal ref where modals will show up
+ */
+export const modalPortalRef = React.createRef<HTMLDivElement>();
```

Comment:
instead of adding `React` in line 32, you can just add `createRef` itself to line 22

Reply:
Good catch!